com.sigmyne.crush.json

```diff
@@ -0,0 +1,49 @@
+{
+    "app-id": "com.sigmyne.crush",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions" : [ "org.freedesktop.Sdk.Extension.openjdk" ],
+    "command": "crush",
+    "finish-args": [
+   	"--share=network",
+       	"--socket=wayland",
+	"--socket=fallback-x11",
+	"--filesystem=host",
+	"--env=PATH=/app/jre/bin:/app/bin:/usr/bin"
+    ],
+    "modules": [
+        {
+            "name": "crush",
+            "buildsystem": "simple",
+            "build-commands": [
+		"/usr/lib/sdk/openjdk/install.sh",
+    		"mkdir -p /app/share/crush",
+		"install -d Install/posix/share /app/",
+		"install -d Install/posix/man /app/share/",
+		"install -D -t /app/share/doc/crush Documentation/*",
+		"install -D -t /app/share/doc/crush copyright",
+		"install -D -t /app/share/doc/crush license.txt",
```

Comment:
Set the mode for installed files.

Reply:
Changed to using `cp -a` instead of `install` for moving files into location.